front/src/Data.elm

```diff
@@ -245,17 +262,12 @@ raspberrySyrup =
 
 raspberryLiqueur : Material
 raspberryLiqueur =
-    { name = "Raspberry liqueur", t = Sweetener }
-
+    { name = "Raspberry liqueur", t = Spirit }
```

Comment:
I think a better typing makes sense. Spirit for 40+ percent, Wine for wine like stuff including vermouth. Liqueur for other things like this? 🤔

Reply:
Yeah, now that there are so many ingredients, more categories definitely make sense! Liqueur, wine, and maybe juice seem like good additional types?